pkg/config/config.go

```diff
@@ -124,6 +124,11 @@ type PluginConfig struct {
 	StreamServerAddress string `toml:"stream_server_address" json:"streamServerAddress"`
 	// StreamServerPort is the port streaming server is listening on.
 	StreamServerPort string `toml:"stream_server_port" json:"streamServerPort"`
+	// StreamIdleTimeout is the maximum time a streaming connection
+	// can be idle before the connection is automatically closed.
+	// The string is in the golang duration format, see:
+	//   https://golang.org/pkg/time/#ParseDuration
+	StreamIdleTimeout string `toml:"stream_idle_timeout" json:"streamIdleTimeout"`
```

Comment:
Add: 
```StreamIdleTimeout: streaming.DefaultConfig.StreamIdleTimeout,```
below in the DefaultConfig function so it will be more obvious to the admin.

Reply:
Good Idea. Done